app/src/main/res/values/strings.xml

```diff
@@ -37,8 +37,8 @@
     <string name="title_activity_double_calibration">Initial Calibration</string>
     <string name="title_activity_calibration_override">Calibration Override</string>
     <string name="pref_header_cloud_storage">Cloud Storage</string>
-    <string name="auto_configure_title">Auto configure</string>
-    <string name="prefs_auto_config_summary">Auto configure using a barcode.</string>
+    <string name="barcode_scanner_title">Barcode Scanner</string>
+    <string name="prefs_barcode_scanner_summary">Import settings from QR or barcode</string>
```

Comment:
As far as I know, the settings are not stored as QR or barcode. The QR code just contains a shared link. Hence, "Import settings using a QR or barcode" or the previous text is more appropriate.

Also, changing the string name removes all translations from Crowdin. A change in the string text just flags the translations as unapproved.

Reply:
My impression was that it was encoded json, based on the [original tool](https://nightscout.github.io/pages/configure/) for sync url setup, but I can take another look at the scanner code.

Reply:
I can confirm that everything is self-contained within the QR codes.

- the xDrip QRs are taking the selected preferences as json, zip-compressing them, and then encoding the data as a base64 string. The "xdpref:" prefix is used to differentiate it from the other QR codes below.
- The Bluejay watch QRs appear to encode their data in lower-level hexadecimals.
- The Nightscout QR codes (for Mongo, MQTT, etc) don't appear to be encoded at all (the QR codes contain straight-up json).

Reply:
> Also, changing the string name removes all translations from Crowdin. A change in the string text just flags the translations as unapproved.

This is still an issue.

Reply:
Is the idea to let this specific feature remain "Auto configure" in other languages for the time being?
I changed other strings as well, and figured if they had to be re-translated anyway it made sense to rework the key-strings while the chance was there.
Side question: am I correct that they just render in English if there's no translation?

For example, it's probably fine to go back and keep using the "copying_settings" translations, but "use_qr_codes_to_transfer_settings_using_settings_auto_configure_feature" would have been a bad one to keep by contrast.

Reply:
I rolled back three string names in the following commit:
0e0c5818caf837a6c3e2caf195dfda7ad37d7301
If we can update the translations when we get the chance, that would be preferable, but I guess it's ok if it takes a bit longer.

Reply:
I've asked for clarification at https://community.crowdin.com/t/rename-of-source-string-identifiers.